src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1278,10 +1308,16 @@
 short-summary: Place the CLI in a waiting state until a condition of a resources is met.
 examples:
   - name: Place the CLI in a waiting state until a condition of a resources is met. (autogenerated)
-    text: az resource wait --exists --ids /subscriptions/{SubID}/resourceGroups/{ResourceGroup}/providers/Microsoft.Web/sites/{WebApp}
+    text: |
+        az resource wait --exists --ids /subscriptions/{SubID}/resourceGroups/{ResourceGroup}/providers/Microsoft.Web/sites/{WebApp}
     crafted: true
   - name: Place the CLI in a waiting state until a condition of a resources is met. (autogenerated)
-    text: az resource wait --exists --ids /subscriptions/{SubID}/resourceGroups/{ResourceGroup}/providers/Microsoft.Web/sites/{WebApp} --include-response-body true
+    text: |
+        az resource wait --exists --ids /subscriptions/{SubID}/resourceGroups/{ResourceGroup}/providers/Microsoft.Web/sites/{WebApp} --include-response-body true
+    crafted: true
+  - name: Place the CLI in a waiting state until a condition of a resources is met. (autogenerated)
+    text: |
+        az resource wait --exists --name MyResource --resource-group MyResourceGroup --resource-type subnets
```

Comment:
The three names here are the same, but the contents of the text are different. Should the name be distinguished?

Reply:
This is the confusion of what Aladdin says "distinct". We need to confirm with them.

Reply:
My bad, it's not "distinct" problem.

It should be "not all parameters affect the title of the example, so when we generate more than one example for a command it’s quite likely to see repeated title".

Aladdin trys to tell user those ways are likely to use comamnds.